.github/workflows/maven.yml

```diff
@@ -26,8 +26,8 @@ jobs:
         distribution: 'temurin'
         cache: 'maven'
     - name: Ensure code is formatted
-      run: mvn com.spotify.fmt:fmt-maven-plugin:check --file kaldb/pom.xml
+      run: mvn -B -Dstyle.color=always com.spotify.fmt:fmt-maven-plugin:check --file kaldb/pom.xml
```

Comment:
fancy!

Reply:
For any Jenkins jobs that are invoked with this we probably should make sure we're doing the batch mode (`-B`) to prevent those progress messages from showing up.